storage/cloudspanner/log_storage.go

```diff
@@ -519,55 +457,51 @@ func (tx *logTX) DequeueLeaves(ctx context.Context, limit int, cutoff time.Time)
 	// moment, FEs queueing entries will be adding them to different buckets
 	// than we're dequeuing from here.
 	now := time.Now().UTC()
-	var rows []spanner.KeySet
+	cfg := tx.getLogStorageConfig()
+	startBucket := int64((now.Unix()+cfg.NumUnseqBuckets/2)%cfg.NumUnseqBuckets) << 8
```

Comment:
Maybe comment this to describe the layout of the bits, which would explain the shifting and masking.

Reply:
ta, done.